test/pre-need/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,121 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts');
+const PageHelpers = require('../e2e/preneed-helpers');
+const testData = require('./schema/maximal-test.json');
+
+const runTest = E2eHelpers.createE2eTest(
+  (client) => {
+    PageHelpers.initApplicationSubmitMock();
+    PageHelpers.initDocumentUploadMock();
+    PageHelpers.initGetCemeteriesMock();
+
+    if (process.env.BUILDTYPE !== 'production') {
```

Comment:
Is this the appropriate flag to use? __BUILDTYPE__ is undefined here.

Reply:
Yeah, as long as it's passing on Jenkins you should be good.